x-pack/plugins/ml/public/application/services/annotations_service.tsx

```diff
@@ -79,3 +79,20 @@ export const annotation$ = new BehaviorSubject<AnnotationState>(null);
 */
 export const annotationsRefresh$ = new BehaviorSubject(Date.now());
 export const annotationsRefreshed = () => annotationsRefresh$.next(Date.now());
+
+export class AnnotationUpdatesService {
+  private _annotation$: BehaviorSubject<AnnotationState> = annotation$;
```

Comment:
You should create a new instance here without statically importing `annotation$`

Reply:
Updated here https://github.com/elastic/kibana/pull/83216/commits/81e20c46cea66e0ca707173eb0c31ef3365f0478